### ISSUE TITLE
Move relation types editor from webforms to Angular

### ISSUE THREAD

**Issue by madsrasmussen** [umbraco/Umbraco-CMS#3436]
Functionality should stay the same.

**Comment by jamescoxhead** [comment]
@madsrasmussen I'd like to pick this one up

**Comment by madsrasmussen** [comment]
@jamescoxhead  🎉 🎉 🎉 Thanks, and have fun :)

**Comment by kjac** [comment]
@jamescoxhead poke me if you need some help on this 😄

**Comment by jamescoxhead** [comment]
Hey @kjac, I've got most of the way there, life got in the way over the last couple of weeks but I should be able to get this done in the next week. Thanks for the offer though!

**Comment by clausjensen** [comment]
@jamescoxhead Hey there :) What's the status on this one? .. is it doable?

**Comment by jamescoxhead** [comment]
@clausjensen Yes doable, just taking a little longer than I expected. Progress is [here](https://github.com/jamescoxhead/Umbraco-CMS/tree/3436-relations-editor), I've got to finish off the create action (half way done) and the delete action.
